### PR TITLE
74970 extend dto validators row version

### PR DESCRIPTION
**Pull request by llykkelarsen**
Validate incoming rowversion to verify that it can be converted to byte array and is thereby valid. Give relevant validation error message.

### CODE REVIEW

File: src/Equinor.Procosys.Preservation.Command/Validators/RowVersionValidator.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+namespace Equinor.Procosys.Preservation.Command.Validators
+{
+    public class RowVersionValidator : IRowVersionValidator
+    {
+        public async Task<bool> IsValid(string rowVersion, CancellationToken cancellationToken) 
+            => TryConvertBase64StringToByteArray(rowVersion, out _);
+
+        private static bool TryConvertBase64StringToByteArray(string input, out byte[] output)
+            {
```
Comment (paal123) [inline src/Equinor.Procosys.Preservation.Command/Validators/RowVersionValidator.cs:13]: fix indentation

File: src/Equinor.Procosys.Preservation.Command/Validators/RowVersionValidator.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+                output = null;
+                try
+                {
+                    output = Convert.FromBase64String(input);
```
Comment (paal123) [inline src/Equinor.Procosys.Preservation.Command/Validators/RowVersionValidator.cs:17]: Can you just remove the whole output variable?

File: src/tests/Equinor.Procosys.Preservation.Command.Tests/Validators/RowVersionValidatorTests.cs [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    [TestClass]
+    public class RowVersionValidatorTests
+    {
+        private RowVersionValidator _dut;
+
+        [TestInitialize]
+        public void SetUp() => _dut = new RowVersionValidator();
+
+        [TestMethod]
+        public async Task IsValid_ValidRowVersion_ReturnsTrue()
```
Comment (paal123) [inline src/tests/Equinor.Procosys.Preservation.Command.Tests/Validators/RowVersionValidatorTests.cs:16]: Add test for null

Comment (paal123) [reply src/tests/Equinor.Procosys.Preservation.Command.Tests/Validators/RowVersionValidatorTests.cs:16]: Add test for string.Empty
